test/runtime/kafka.go

```diff
@@ -139,4 +139,38 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		res = vm.Exec(consumer(disallowTopic, MaxMessages))
 		res.ExpectFail("Kafka consumer can access to disallowTopic")
 	})
+
+	It("Kafka Policy Role Ingress", func() {
+		_, err := vm.PolicyImportAndWait(vm.GetFullPath("Policies-kafka-Role.json"), 300)
+		Expect(err).Should(BeNil())
```

Comment:
Add descriptions to all Expects. If the test fails we will only see something like:
```
line 145: Expected nil, retrieved "foo"
```
with a description we will know
```
line 145: Expected nil, retrieved "foo"... While importing `Policies-kafka-Role.json`
```

Reply:
done